src/Controls/src/Core/DatePicker.cs

```diff
@@ -37,11 +39,18 @@ public DatePicker()
 		{
 			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<DatePicker>>(() => new PlatformConfigurationRegistry<DatePicker>(this));
 		}
-
+		public DateTime? SelectedDate
+		{
+			get { return (DateTime)GetValue(SelectedDateProperty); }
+			set { SetValue(SelectedDateProperty, value); }
+		}
 		public DateTime Date
```

Comment:
I think since MAUI is new, we can probably just change the existing property.

Reply:
I would also like that, but I wonder if Microsoft is concerned with maintaining back-compat for people migrating from Xamarin.Forms...

Reply:
We are, but not at this level I don't think. We already have changed a fair bit, and making this nullable is probably not as breaking as other things we have done.

Reply:
Hi Yes you are correct @KPixel they tend not to like us messing with existing properties, that is why I stayed with selected date.

Reply:
If @mattleibow says it is ok, then lets just change the Date property. That's a lot simpler :)

Is it also ok to change to `System.DateOnly`? (I guess that's more "break-y")

Reply:
I think we would run the risk of braking stuff if we changed the Date property people have built ui around the fact it may never be null. @mattleibow can you double check of someone if that would be ok to do so?

Reply:
I can re do the pull request to only change the Date to be null and thus not adding a new property.

Reply:
@mattleibow How would I handle the issue here,
Obv as we are making the date null here what should we do to get a blank picker.
This is why I was saying to @KPixel it seems to require a valid date for that part?

			var date = VirtualView.Date;
			ShowPickerDialog(date.Year, date?.Month, date?.Day);

Reply:
@mattleibow  We are having lots of issue with NSDate aprently the code base is not liking the fact we changed to a null Date, when am attempting it here am having issues with TotalSeconds it doesnt appear to like the fact we introduced a null.

	public static NSDate ToNSDate(this DateTime date)
		{	
		   return NSDate.FromTimeIntervalSinceReferenceDate((date - ReferenceDate).TotalSeconds);			
		}

Reply:
Interesting... I wonder. @PureWeen, what are our thoughts on this new type?

Reply:
We already changed Color to have null by default so i think it's fine.